### DISCUSSION TITLE
Age password caching

### DISCUSSION THREAD

**Discussion by conor-sometimes** [twpayne/chezmoi#3178] [Q&A] [answered]
Hi I use the same password when I encrypt files, when I run `chezmoi -v apply` and it asks my for my `age` password to decrypt files, it asks 3 times decrypting each file individually.

Is there any way to cache the password so I can enter a password once and decrypt all files.

Apologies if it's been asked before, I couldn't see anything in the discussions when I looked through.

**Comment by twpayne** [comment, marked as answer]
See [this entry in the user guide](https://www.chezmoi.io/user-guide/frequently-asked-questions/encryption/#how-do-i-configure-chezmoi-to-encrypt-files-but-only-request-a-passphrase-the-first-time-chezmoi-init-is-run).

**Reply by conor-sometimes** [reply]
Wow, I think I must be blind sometimes, sorry about that!
Thanks!
